src/docs/release/breaking-changes/add-currentAutofillScope-to-TextInputClient.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Add currentAutofillScope to TextInputClient
+description: A new getter`TextInputClient.currentAutofillScope` was added to the `TextInputClient` interface for autofill support.
+---
+
+## Summary
+
+A new getter`TextInputClient.currentAutofillScope` was added to the
+`TextInputClient` interface.
+
```

Comment:
maybe also mention the context in which you might have your own TextInputClient subclass.